components/notion/actions/common/constants.mjs

```diff
@@ -0,0 +1,181 @@
+const ICON_TYPES = [
+  "text",
+  "emoji",
+];
+
+const COVER_TYPES = [
+  "external",
+  "file",
+];
+
+/**
+ * Notion block types, containing:
+ *   - Notion key string
+ *   - props
+ *   - additionalProps
+ */
+const BLOCK_TYPES = {
+  paragraph: {
+    key: "paragraph",
```

Comment:
[This check](https://github.com/PipedreamHQ/pipedream/runs/5514347184?check_suite_focus=true) fails because the [findBadKeys.js](https://github.com/PipedreamHQ/pipedream/blob/master/scripts/findBadKeys.js) script checks that the first `key` property in a JS file starts with `${nameSlug}-`.

As a workaround, I think you could change the name of this property (e.g., from `key` to `name`).